File: tervezes/uml_gen/uml_generator/Bear.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VeryGoodViroGame
{
    public class Bear : Agent
    {
        public Bear(): base()
        {
            //strategy = new MSBear();
        }

        public override void Apply(Virologist source, Virologist target)
        {
            base.Apply(source, target);
        }

        
        public override void Step()
        {
        }
    }
}

File: VeryGoodViroGame/src/VeryGoodViroGame/Agent/Bear.java
package VeryGoodViroGame.Agent;

import VeryGoodViroGame.DrawableComponent;
import VeryGoodViroGame.MoveStrategy.MSBear;

public class Bear extends Agent
{
   
    public Bear()
    {
        super();
        strategy = new MSBear();
    }
    
    @Override
    public void Step()
    {
        //Do not lower counter
    }
    @Override
    public String GetDrawString()
    {
        return "bear";
    }
}
